src/components/home/home.styles.js

```diff
@@ -0,0 +1,238 @@
+    backgroundImage: 'url(./home-background.svg)',
+    backgroundPosition: 'bottom',
+    backgroundSize: 'cover',
+    display: 'flex',
+    '& > div': {
+      marginRight: 'auto',
+      marginLeft: 0
+    },
+    '& > div a': {
+      marginLeft: 0
+    }
+  },
+  section1Image1: {
+    position: 'absolute',
+    top: 0,
+    right: 0,
+    width: '100vw'
+  },
+  section2: {
+    display: 'flex',
+    flexDirection: 'column',
+    '& >:first-child': {
+      textAlign: 'center',
+      marginBottom: '106px'
+    },
+    '& >:last-child': {
+      margin: '124px auto 0',
+      height: '42px'
+    },
+    '& >div div': {
+      textAlign: 'center'
+    }
+  },
+  section2Image1: {
+    width: '142px',
+    height: '147px'
+  },
+  section2Image2: {
+    width: '162px',
+    height: '147px'
+  },
+  section2Image3: {
+    width: '141px',
+    height: '147px'
+  },
+  section2Text: {
+    width: '245px',
+    margin: '50px auto 0',
+    fontWeight: '500',
+    fontSize: '23px',
+    lineHeight: '35px'
+  },
+  section3: {
+    backgroundColor: '#f6f7fa'
+  },
+  section3Text: {
+    color: '#778499',
+    lineHeight: '29px',
+    margin: '24px 0'
+  },
+  section4: {
+    '& >div': {
+      paddingTop: '48px'
+    },
+    '& $col2': { // TODO refactor all inheritance
+      color: '#778499',
+      lineHeight: '29px'
+    },
+    '& $col3': {
+      marginTop: 0,
+      marginRight: '60px',
+      '&:last-child': {
+        marginRight: 0
+      }
+    }
+  },
+  section5: {
+    backgroundColor: '#f6f7fa'
+  },
+  section6: {
+    '& >div': {
+      paddingTop: '48px'
+    },
+    '& $col2': { // TODO refactor all inheritance
+      color: '#778499',
+      lineHeight: '29px'
+    },
+    '& $col3': {
+      marginTop: 0,
+      marginRight: '60px',
+      '&:last-child': {
+        marginRight: 0
+      }
+    },
+    '& h3': {
+      color: '#000',
+      fontSize: '24px'
+    },
+    '& a': {
+      marginTop: '26px'
+    },
+    '& $divider': {
+      width: '50%',
+      marginLeft: 'auto'
+    }
+  },
+  section7: {
+    backgroundColor: '#f6f7fa',
+    '& >div': {
+      paddingTop: '32px',
+      '&:last-child': {
+        width: '50%',
+        marginLeft: 'auto'
+      }
+    },
+    '& $col2': { // TODO refactor all inheritance
+      color: '#778499',
+      lineHeight: '29px'
+    },
+    '& $col3': {
+      marginTop: 0,
+      marginRight: '60px',
+      '&:last-child': {
+        marginRight: 0
+      }
+    },
+    '& h3': {
+      color: '#000',
+      fontSize: '24px'
+    },
+    '& a': {
+      marginTop: '26px'
+    },
+    '& $divider': {
+      width: '50%',
+      marginLeft: 'auto'
+    }
+  },
+  row: {
+    display: 'flex'
+  },
+  col2: {
+    width: '50%',
+    marginLeft: 'auto'
+  },
+  col3: {
+    margin: 'auto',
+    // textAlign: 'center',
+    width: '33%'
+  },
+  buttonLink: {
+    padding: '26px 28px',
+    fontWeight: '700',
+    // padding: `${theme.spacing(3)}px ${theme.spacing(3.5)}px`,
```

Comment:
Why all this comments?

Reply:
Forgot to remove from when theme was not working, I'll update.